packages/iov-bns-governance/src/integrationtests/proposals.spec.ts

```diff
@@ -25,6 +26,7 @@ import { ReadonlyDate } from "readonly-date";
 import { CommitteeId } from "../committees";
 import { Governor, GovernorOptions } from "../governor";
 import { ProposalOptions, ProposalType } from "../proposals";
+import { triggerAsyncId } from "async_hooks";
```

Comment:
?

Reply:
VSCode doesn't understand my habit of writing code that references variables I need before writing the code that declares them.